CodeGeneration/Sources/Utils/CodeGenerationFormat.swift

```diff
@@ -17,11 +17,42 @@ import SwiftSyntax
 public class CodeGenerationFormat: BasicFormat {
   public override var indentation: TriviaPiece { .spaces(indentationLevel * 2) }
 
-  public func ensuringTwoLeadingNewlines<NodeType: SyntaxProtocol>(node: NodeType) -> NodeType {
-    if node.leadingTrivia?.first?.isNewline ?? false {
-      return node.with(\.leadingTrivia, indentedNewline + (node.leadingTrivia ?? []))
+  public override func visit(_ node: ArrayElementListSyntax) -> ArrayElementListSyntax {
+    let children = node.children(viewMode: .all)
+    // Short array literals are presented on one line, list each element on a different line.
+    if children.count > 2 {
```

Comment:
One last question: Why did you choose 2 for arrays and dictionaries but 3 for function parameters? If there’s a reason, I’m totally fine with it, if it’s arbitrary I think it would be nice to keep them the same so the next person who looks at this doesn’t wonder why there’s a difference.

Reply:
I think it reads better. But I don't have strong options here. 
I actually rather would make a newline for all arrays + dictionaries

Reply:
Changed all to 3 for now. 

We can change later if we think it will improve the readability

Reply:
I’m not saying I want 2 or 3. It’s totally fine to mix and match. I just wanted to make sure you deliberately chose those values if they are different.